// @angular/cdk/stepper/typings/index.ngfactory.js
"use strict";
/**
 * @fileoverview This file was generated by the Angular template compiler. Do not edit.
 *
 * @suppress {suspiciousCode,uselessCode,missingProperties,missingOverride,checkTypes}
 * tslint:disable
 */ 
Object.defineProperty(exports, "__esModule", { value: true });
var i0 = require("@angular/core");
var i1 = require("@angular/cdk/stepper");
var i2 = require("@angular/common");
var i3 = require("@angular/cdk/bidi");
var CdkStepperModuleNgFactory = i0.ɵcmf(i1.CdkStepperModule, [], function (_l) { return i0.ɵmod([i0.ɵmpd(512, i0.ComponentFactoryResolver, i0.ɵCodegenComponentFactoryResolver, [[8, []], [3, i0.ComponentFactoryResolver], i0.NgModuleRef]), i0.ɵmpd(4608, i2.NgLocalization, i2.NgLocaleLocalization, [i0.LOCALE_ID, [2, i2.ɵangular_packages_common_common_a]]), i0.ɵmpd(1073742336, i3.BidiModule, i3.BidiModule, []), i0.ɵmpd(1073742336, i2.CommonModule, i2.CommonModule, []), i0.ɵmpd(1073742336, i1.CdkStepperModule, i1.CdkStepperModule, [])]); });
exports.CdkStepperModuleNgFactory = CdkStepperModuleNgFactory;
var styles_CdkStep = [];
var RenderType_CdkStep = i0.ɵcrt({ encapsulation: 2, styles: styles_CdkStep, data: {} });
exports.RenderType_CdkStep = RenderType_CdkStep;
function View_CdkStep_1(_l) { return i0.ɵvid(0, [i0.ɵncd(null, 0), (_l()(), i0.ɵand(0, null, null, 0))], null, null); }
function View_CdkStep_0(_l) { return i0.ɵvid(2, [i0.ɵqud(402653184, 1, { content: 0 }), (_l()(), i0.ɵand(0, [[1, 2]], null, 0, null, View_CdkStep_1))], null, null); }
exports.View_CdkStep_0 = View_CdkStep_0;
function View_CdkStep_Host_0(_l) { return i0.ɵvid(0, [(_l()(), i0.ɵeld(0, 0, null, null, 2, "cdk-step", [], null, null, null, View_CdkStep_0, RenderType_CdkStep)), i0.ɵdid(1, 573440, null, 1, i1.CdkStep, [i1.CdkStepper, [2, i1.STEPPER_GLOBAL_OPTIONS]], null, null), i0.ɵqud(335544320, 1, { stepLabel: 0 })], null, null); }
exports.View_CdkStep_Host_0 = View_CdkStep_Host_0;
var CdkStepNgFactory = i0.ɵccf("cdk-step", i1.CdkStep, View_CdkStep_Host_0, { stepControl: "stepControl", label: "label", errorMessage: "errorMessage", ariaLabel: "aria-label", ariaLabelledby: "aria-labelledby", state: "state", editable: "editable", optional: "optional", completed: "completed", hasError: "hasError" }, {}, ["*"]);
exports.CdkStepNgFactory = CdkStepNgFactory;
